fastapi-google-stt/app/services/log_script_service.py

```
@@ -0,0 +1,16 @@
+import io
+from app.dto.message_request import MessageRequest
+
+speacker = None
+file = None
+
+def logger(data: MessageRequest, x_user_id: str = None):
+    """
+    로그를 파일에 저장하는 함수
+    """
+
+    if speacker is None and file is None:
```

Ruff (0.8.2)

Local variable `speacker` referenced before assignment

(F823)